### PR TITLE
Remove _MM and slightly change DecayTreeFitter lesson

### PR DESCRIPTION
**Pull request by saschastahl**
- I changed the link to second analysis steps to a DaVinci tutorial as I did not find a topic about DecayTreeFitter
- Removed the use of _MM, the graphic still needs to be adapted. Added a callout box, trying to explain the difference.

### CODE REVIEW

File: decay-tree-fitter.md [hunk 1]
```diff
@@ -47,11 +47,11 @@ When using the `DecayTreeFitter` in a `DecayTreeTuple`, all the variables create
 If the daughters are not stable particles and decay further, the daughters of the daughters have no new variables associated to them by default.
 Since in many cases this information might be useful, there is an option to store the information from those tracks
 ```python
-dtt.Dstar.ConsD.UpdateDaughters = True
+dtt.Dstar.ConsD.UpdateDaughters = Truedecay-tree-fitter.md
```
Comment (rmatev) [inline decay-tree-fitter.md]: typo?

File: decay-tree-fitter.md [hunk 2]
```diff
@@ -63,10 +63,14 @@ root -l DVntuple.root
...
 DecayTree->StartViewer()
 ```
-Plotting the raw mass of the D* (without the fit) `Dstar_MM` you should see a broad signal around 2 GeV:
+Plotting the raw mass of the D* (without the fit) `Dstar_M` you should see a broad signal around 2 GeV:
 
 <img src="./img/DstarRaw.png" alt="Dstar raw" style="width: 500px;"/>
 
+> ## Which mass variable to use {.callout}
+> In many ntuples you also find a mass variable called `_MM`. This confusingly refers to measured mass. However, it is usually better to use `_M`. `_MM` is the sum of the 4-momenta of the final state particles at the fitted vertex position, but not the result of the actual vertex fit.
+
```
Comment (alexpearce) [inline decay-tree-fitter.md:72]: To be more explicit, maybe “…sum of the 4-momenta of the final state particles extrapolated back to the fitted vertex position“. To get them thinking about the extrapolation itself.

**Comment by saschastahl** [comment]
@alexpearce Should have opened a new pull request. I changed the links in lesson 13 from svn to gitlab.

**Comment by alexpearce** [comment]
Have you checked that all SVN links are gone now?

**Comment by saschastahl** [comment]
At least in this lesson.

**Comment by alexpearce** [comment]
LGTM 👍 Thanks a lot, this is Good Advice™.

**Pull request merged**